boards/pjrc/teensy4/Kconfig.teensyMM

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2020, Bernhard Kraemer
```

Comment:
lowercase filenames

Reply:
I renamed the ones with MM in them to mm
But I did not change the K to k at the start of all of the Kconfig.xxx file names